src/constants/times.js

```diff
@@ -0,0 +1,14 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export const MILLISECONDS_PER_HOUR = 60 * 60 * 1000;
+export const SEVEN_DAYS_IN_HOURS = 7 * 24;
```

Comment:
I like the general idea here, but I feel like it would be much more flexible/reusable as a utility called `convertTime.js` that looks like this:

```
export const MILLISECOND = 1;
export const SECOND = MILLISECOND * 1000;
export const MINUTE = SECOND * 60;
export const HOUR = MINUTE * 60;
export const DAY = HOUR * 24;
export const WEEK = DAY * 7;
// etc...

export default (fromUnit, toUnit, amount) => (fromUnit * amount) / toUnit;
};
```